test/db/formats/mach0/objc

```diff
@@ -63,7 +63,7 @@ NAME=objc method flags (icj)
 FILE=bins/mach0/TestMethods
 CMDS=icj~{[0].methods[2].flags[0]}
 EXPECT=<<EOF
-class
+"class"
```

Comment:
would it make sense to unwrap the values?

Reply:
It would need changing the PJ API, `pj_s()` function in particular, which wraps all strings in quotes. I can add a new function though, `pj_S()`, for example, to print escaped string but without quotes. Sounds good?

Reply:
@ret2libc yes, they are: https://github.com/rizinorg/rizin/blob/dev/librz/util/pj.c#L201

Reply:
I mean in the old code, the one in sdb. sdb_json_get_str

Reply:
Fixed in https://github.com/rizinorg/rizin/pull/1118/commits/111fac88a45857ca657267968e02776790b63381